pkg/innerring/invoke/enhanced.go

```diff
@@ -54,3 +57,18 @@ func NewNoFeeAuditClient(cli *client.Client, contract util.Uint160) (*auditWrapp
 
 	return auditWrapper.WrapClient(audit.New(staticClient)), nil
 }
+
+// NewNoFeeBalanceClient creates wrapper to work with Balance contract.
+func NewNoFeeBalanceClient(cli *client.Client, contract util.Uint160) (*balanceWrapper.Wrapper, error) {
+	staticClient, err := client.NewStatic(cli, contract, readOnlyFee)
```

Comment:
I looked through it again and it seems like there is no point in `NoFee` clients at all, since reading operations are free no matter of fee specified in static client. 

In #355 I propose to use `extraFee` in all these clients so it should fix this issue. Let's sync based on this commit. https://github.com/nspcc-dev/neofs-node/pull/355/commits/d244aabec5987a3753f7096b40c11b2b244a8002

What do you think? @cthulhu-rider

Reply:
Okay, I'll rename `NewNoFeeBalanceClient` -> `NewBalanceClient` and replace `readOnlyFee` with `extraFee`.

Reply:
Rdy